### PR TITLE
Doc update with zenodo data

### PR DESCRIPTION
**Pull request by blowekamp**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -33,29 +33,28 @@ Installation
 ------------
 
 
-The Python module is distributed as a `wheel`_ binary package. Download the latest tagged release from the
-`Github Releases`_ page. Then install::
+The Python module is distributed as a `wheel`_ binary package. To install the latest tagged release downloaded from
```
Comment (zivy) [inline README.rst:36]: Remove the word downloaded: "To install the latest tagged release downloaded from..." -> "To install the latest tagged release from..."

File: README.rst [hunk 2]
```diff
@@ -64,22 +63,25 @@ The following examples uses CD4 marker channel extracted from the "IBEX4_spleen"
...
+Then apply the registration transform by resampling all channels of the the input images onto panel 2::
 
- python -m sitkibex resample IBEX4B_Panel2_CD4_AF594.nrrd IBEX4B_Panel1_CD4_AF594.nrrd tx_p2_to_p1.txt \
-        -o IBEX4B_onto_p2_Panel1_CD4_AF594.nrrd
- python -m sitkibex resample IBEX4B_Panel3_CD4_AF594.nrrd IBEX4B_Panel1_CD4_AF594.nrrd tx_p3_to_p1.txt \
-        -o IBEX4B_onto_p2_Panel3_CD4_AF594.nrrd
+ python -m sitkibex resample "spleen_panel2.nrrd@CD4 AF594" spleen_panel2.nrrd tx_p2_to_p1.txt \
+        -o spleen_onto_p2_panel1.nrrd
+ python -m sitkibex resample spleen_panel2.nrrd@CD4 AF594" spleen_panel3.nrrd@CD4 tx_p2_to_p3.txt \
+        -o speen_onto_p2_panel3.nrrd
```
Comment (zivy) [inline README.rst:84]: speen_onto... -> spleen_onto...

**Review by zivy — APPROVED** [review]

File: README.rst (hunk 1 above)
Comment (blowekamp) [reply README.rst]: done

File: README.rst (hunk 2 above)
Comment (blowekamp) [reply README.rst]: done

**Pull request merged**